bindings_ffi/src/lib.rs

```diff
@@ -65,8 +70,21 @@ pub async fn create_client(
         .await
         .map_err(|e| stringify_error_chain(&e))?;
 
+    let store = match db {
+        Some(path) => {
+            info!("Using persistent storage: {} ", path);
+            EncryptedMessageStore::new_unencrypted(StorageOption::Persistent(path))
+        }
+
+        None => {
+            info!("Using ephemeral store");
+            EncryptedMessageStore::new(StorageOption::Ephemeral, static_enc_key())
+        }
+    };
+
     let mut xmtp_client: RustXmtpClient = xmtp::ClientBuilder::new(inbox_owner.into())
         .api_client(api_client)
+        .store(store.unwrap())
```

Comment:
Unwrap is bad, right?

Reply:
Unwrap will panic if an error is found. Would be nicer to have a `.map_err` and convert the error to a `GenericError`, which will then get returned to the native iOS/Android caller.

Reply:
Cool, fixed in https://github.com/xmtp/libxmtp/pull/219/commits/9ee92904cfd5f838913c1d0efa5c6a5faed539a5.